pkg/controller/nodeipam/ipam/cloud_cidr_allocator_metrics.go

```diff
@@ -20,6 +20,15 @@ var (
 		},
 		[]string{"network"},
 	)
+	cloudCidrAllocatorDroppedNodes = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      nodeIpamSubsystem,
+			Name:           "cloudcidrallocator_dropped_nodes",
+			Help:           "Number of times nodes have errored enough to be dropped from the cloudCidrAllocator workqueue",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"node"},
```

Comment:
I had to look into this further as I didn't fully understand cardinality issues.

Yes, youre right. This will cause issues at high amounts of nodes. GNP objects should be fine because there should never be thousands of them.